### PR TITLE
Encrypt sms code in cookie

### PR DESCRIPTION
**Pull request by richtier**
signed cookies are not encrypted, and unhashing them is super easy:
```
>> from django.core import signing
>> from django.contrib.sessions.backends.signed_cookies import SessionStore
>> signing.loads(
	cookie_value,
	serializer=SessionStore.serializer,
	salt='django.contrib.sessions.backends.signed_cookies'
)
```

image

this demonstrates the need to encrypt the sms code.

with bad code:

image

with good code:
image

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -6,6 +6,7 @@ raven==5.19.0
 requests==2.10.0
 whitenoise==3.1
 phonenumberslite==7.7.4
+pycrypto==2.6.1
```
Comment (konradko) [inline requirements.txt]: is this used anywhere?

**Comment by konradko** [comment]
@richtier don't you need to know the `SECRET_KEY` to load the signed cookie?

File: requirements.txt (hunk 1 above)
Comment (konradko) [inline requirements.txt]: seems like a leftover

Comment (richtier) [inline requirements.txt]: ah yes, I changed my approach. was going to roll my own crypto...

**Comment by richtier** [comment]
@konradko signed cookies prevent tampering, they do not prevent viewing the value. see the PR description for how easy it is to unhash a signed cookie.

**Pull request merged**